web/controllers/user_controller.ex

```diff
@@ -34,6 +35,21 @@ defmodule CodeCorps.UserController do
     end
   end
 
+  def github_connect(conn, code) do
+    current_user = Guardian.Plug.current_resource(conn)
+    with {:ok, user} <- Github.connect(current_user, code)
+    do
+      # {:ok, user}
```

Comment:
@npendery I'm afraid I assumed too much about ja_resource there. You'll have to do a `conn |> render` manually as you would in a ja_resourceless controller.